app/src/main/java/com/elavon/converge/config/ConfigLoader.java

```diff
@@ -12,28 +12,35 @@
 
 public class ConfigLoader {
 
+    private Gson gson;
     private final Resources resources;
     private final Config.Environment environment;
 
     public ConfigLoader(final Resources resources, final Config.Environment environment) {
+        this.gson = new Gson();
         this.resources = resources;
         this.environment = environment;
     }
 
     public Config load() {
-        final int config;
+        final int configResource;
         switch (environment) {
             case LIVE:
-                config = R.raw.config_live;
+                configResource = R.raw.config_live;
                 break;
             case TEST:
             default:
-                config = R.raw.config_test;
+                configResource = R.raw.config_test;
                 break;
         }
 
-        final String configText = readFile(resources.openRawResource(config));
-        return new Gson().fromJson(configText, Config.class);
+        final String configText = readFile(resources.openRawResource(configResource));
+        final String credentialText = readFile(resources.openRawResource(R.raw.credential));
```

Comment:
loading from `R.raw.credential` so we don't mix it with config file. Again, this is temporary.